tests/drivers/i2s/i2s_api/src/test_i2s_loopback.c

```diff
@@ -335,10 +218,20 @@ void test_i2s_rx_sync_start(void)
 
 	/* Prefill TX queue */
 	for (int n = 0; n < NUM_TX_BLOCKS; n++) {
-		ret = k_mem_slab_alloc(&tx_0_mem_slab, &tx_block, K_FOREVER);
-		zassert_equal(ret, TC_PASS, NULL);
-		fill_buf_const((u16_t *)tx_block, 1, 2);
-		ret = i2s_write(dev_i2s, tx_block, BLOCK_SIZE);
+		if (test_priv_mode == TEST_SUPERVISOR_MODE) {
```

Comment:
I wonder if we should use the same code for supervisor and user mode, ie always calling `i2s_buf_write` instead of allocating a memory block with `k_mem_slab_alloc` followed by `i2s_write`.

Reply:
ok, done